packages/ovee/src/core/App.ts

```diff
@@ -145,12 +146,25 @@ export default class App {
 		Object.values(this.modules).forEach(module => module.destroy());
 	}
 
-	getModule(name: string): Module {
+	getModule<M extends ModuleClass = ModuleClass>(module: M | string): InstanceType<M> {
+		let name: string;
+		console.log(Module);
```

Comment:
👮 Sir, please step out of the vehicle, and I will read you your rights

Reply:
🥖